arch/arm/src/common/arm_assert.c

```diff
@@ -172,44 +175,66 @@ static void arm_dump_task(struct tcb_s *tcb, void *arg)
     }
 #endif
 
+#ifndef CONFIG_DISABLE_PTHREAD
+  if ((tcb->flags & TCB_FLAG_TTYPE_MASK) == TCB_FLAG_TTYPE_PTHREAD)
+    {
+      FAR struct pthread_tcb_s *ptcb = (FAR struct pthread_tcb_s *)tcb;
+
+      snprintf(args, sizeof(args), "%p ", ptcb->arg);
+    }
+  else
+#endif
+    {
+      FAR char **argv;
+      size_t npos = 0;
+
+      for (argv = tcb->group->tg_info->argv + 1; *argv; argv++)
+        {
+          npos += strlcpy(args + npos, *argv, sizeof(args) - npos);
+          npos += strlcpy(args + npos, " ", sizeof(args) - npos);
```

Comment:
why not to use `snprintf` with `"%s "` here as well?

Reply:
Done.